chapters/syntax.tex

```diff
@@ -57,10 +57,7 @@ \section{Lexical conventions}\label{lexical-conventions}
   Modelica uses the same comment syntax as C++ and Java (i.e., \lstinline!//! signals the start of a line comment and \lstinline!/* $\ldots$ */! is a multi-line comment); comments may contain any Unicode character.
   Modelica also has structured comments in the form of annotations and string comments.
 \item
-  Each \lstinline[language=grammar]!description-string! or string in annotations (that is, a \lstinline[language=grammar]!STRING! with production \lstinline[language=grammar]!annotation-clause! in the grammar) may contain any member of the Unicode character set.
-  All other strings have to contain only the subset of Unicode characters identical with the 7-bit US-ASCII character set.
-  \begin{nonnormative}
-  As a consequence, operators like `\lstinline!>!' or `\lstinline!<!', and external functions only operate on ASCII strings and not on Unicode-strings.
+ \begin{nonnormative}
   Within a \lstinline[language=grammar]!description-string! the tags \lstinline!<HTML>! and \lstinline!</HTML>! or \lstinline!<html>! and \lstinline!</html>! define optionally begin and end of content that is HTML encoded.
```

Comment:
I am not sure what happened. I remember writing a comment on this line but do not see it. So here it goes again.

This bullet item now consists just out of the non-normative part. It looks a bit odd. Also the sentence is not grammatically correct. We discussed it here: https://github.com/modelica/ModelicaSpecification/pull/3045/files#r755270942 and there was a separate issue #3051. Now that is even more odd should we remove it from here completely? @henrikt-ma, you are more familiar with how far #3051 progressed.

Reply:
We didn't make much progress at all on that one, but that is where we should be discussing whether to remove it completely or not.

Reply:
I removed the entire item, instead of correcting the grammar.
To me it originally made sense in this context. When we didn't have UTF-8 in annotations it made sense to explain that one could use html - and thus implicitly html-encoded characters, like `&Ouml; &euro;` - since we have seen a number of non-ASCII characters in the documentation. 

But currently, I don't see a problem with removing it, since we still have the corresponding text in the annotation-chapter.